dev/tests/integration/testsuite/Magento/Payment/Helper/DataTest.php

```diff
@@ -21,4 +23,36 @@ public function testGetInfoBlock()
         $result = $helper->getInfoBlock($paymentInfo);
         $this->assertInstanceOf(\Magento\OfflinePayments\Block\Info\Checkmo::class, $result);
     }
+
+    /**
+     * Checking if payment available in getPaymentMethodList() with set active to 0.
+     *
+     * @magentoConfigFixture default_store payment/checkmo/active 0
+     */
+    public function testGetPaymentMethodListWithInActivePayments()
+    {
+        /** @var \Magento\Payment\Helper\Data $helper */
+        $helper = Bootstrap::getObjectManager()->get(\Magento\Payment\Helper\Data::class);
+        $paymentMethodsList =  $helper->getPaymentMethodList();
+        $this->assertArrayHasKey('cashondelivery',$paymentMethodsList);
```

Comment:
Why do we need to check this one?

Reply:
This payment have empty active state, so at this point we check if payment available if they not have any settings.

Reply:
We do not control this payment method in test so I would recommend to remove such assertion.

Reply:
okay